code/datums/music_tracks/elibao.dm

```diff
@@ -0,0 +1,16 @@
+/music_track/elibao
+	artist = "Earthcrusher"
+	title = "every light is blinking at once"
+	song = 'sound/music/elibao.ogg'
+	license = /decl/license/cc_by_nc_sa_3_0
```

Comment:
the license listed here is different from the one in the comment, please investigate this one thoroughly

Reply:
It's NC according to the soundcloud, comment updated